index.html

```diff
@@ -39,131 +39,158 @@ <h3 class="cover-links">LET'S CONNECT</h3>
+          <div class="clients">
+            <h4>CANOPY</h4>
+            <img class="img-clients" src="images/works/Counter.svg" alt="">
+            <h4 style="color: #6b778c">Back End Dev</h4>
+            <img class="img-clients" src="images/works/Counter.svg" alt="">
+            <h4 style="color: #6b778c">2015</h4>
+          </div>
+            <p class="highlights">
+              A daily selection of privately personalized reads no accounts for
+              sign-ups required.
+            </p> 
+    
+            <ul class="taggs" style="list-style: none;">
+              <li><button class="tag1">html</button></li>
+              <li><button class="tag2">css</button></li>
+              <li><button class="tag3">javascript</button></li>
+            </ul>  
+    
+            <button class="project-detail">See Project</button>
+        </div>
+      </section>
+    </section>
+<!-- About myself section -->
+    <section id="about-me" class="about-myself">
+      <div class="about-myself-content">
+          <h2 class="about-myself-title">
+            About <br />
+            Myself
+          </h2>
+          <p class="myself-content">
+            Hello I am a software develper. I can help you build a product, feature
+            or website. Look through some of my work and expereince. If you like
+            what you see and have a project you need coder, don't husitate to
+            contact me.
+          </p>
+          <h3 class="links">LET'S CONNECT</h3>
+          <ul class="socials">
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#" class="fa fa-linkedin"></a></li>
+            <li><a href="#" class="fa fa-google"></a></li>
+            <li><a href="#" class="fa fa-github"></a></li>
+            <li><a href="#" class="fa fa-javascript"></a></li>
+          </ul>
+          <button class="resume">Get My Resume</button>
+      </div>
+      <div class="about-myself-languages">
+          <h3 class="languages">Languages</h3>
+          <ul class="framework-skills">
+            <li><img src="images/skills/html.svg" alt="HTML skill" /></li>
+            <li><img src="images/skills/css.svg" alt="CSS skill" /></li>
+            <li><img src="images/skills/js.svg" alt="Javascript skill" /></li>
+          </ul>
+          <ul class="frameworks" style="list-style: none;">
+            <li class="framework1">Frameworks</li>
+            <li class="greater-tag">></li>
+          </ul>
+          <hr>
+          <ul class="frameworks" style="list-style: none;">
+            <li class="framework1">Skills</li>
+            <li class="greater-tag">></li>
+          </ul>
+          <hr>
       </div>
-        <p class="highlights">
-          A daily selection of privately personalized reads no accounts for
-          sign-ups required.
+      <!-- Contact page -->
+      <div id="my-contact" class="contact">
```

Comment:
- [ ] Kindly remove the padding around the contact section because the width of the contact section is the same as the viewport width. Kindly remove it so it matches the figma design 👍🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212974605-7c415453-e4e6-485d-99f7-fa7d5f18a153.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212974750-4b634931-279f-466c-9f2a-24e93b205b25.png)